src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -170,53 +163,31 @@ export const Breadcrumbs = React.forwardRef(function Breadcrumbs(
         }
         const hiddenItems = breadcrumbs.slice(0, -endItems);
         const menuItem = (
-            <BreadcrumbItem itemType="menu">
-                <DropdownMenu
-                    items={hiddenItems.map((el, index) => {
-                        return {
-                            ...el.props,
-                            text: el.props.children,
-                            disabled: props.disabled,
-                            items: [],
-                            action: (event) => {
+            <BreadcrumbsDropdownMenu
+                disabled={props.disabled}
+                popupPlacement={props.popupPlacement}
+                popupStyle={props.popupStyle}
+                component={BreadcrumbsDropdownMenu}
+            >
+                {hiddenItems.map((child, index) => {
+                    const Component = child.props.component ?? BreadcrumbsItemView;
+                    return (
+                        <Component
+                            {...child.props}
+                            index={index}
+                            key={child.key}
+                            disabled={props.disabled || child.props.disabled}
+                            onAction={() => {
```

Comment:
Shouldn't `onAction` to be listed in the `BreadcrumbsItemProps`?

Reply:
onAction returns the key of the element that triggered the action.